Source/Core/DolphinQt2/GameList/GameTracker.cpp

```diff
@@ -72,23 +72,63 @@ void GameTracker::UpdateDirectory(const QString& dir)
   while (it.hasNext())
   {
     QString path = QFileInfo(it.next()).canonicalFilePath();
+    auto& tracked_file = m_tracked_files[path];
```

Comment:
This should probably also be used in the else case if it's in this scope (or be moved into the inner if statement scope if that would cause issues).